### PR TITLE
Create integration-IronPort

### PR DESCRIPTION
**Pull request by shaniacht1**
fixes https://github.com/demisto/server/issues/6172

### CODE REVIEW

**Comment by asafshen** [comment]
@shaniacht1 please ensure that all commands & arguments have description (will be checked automatically when we will push schema validation), also those with predefine values

File: Integrations/integration-IronPort.yml [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Port
```
Comment (Spivaka) [inline Integrations/integration-IronPort.yml]: port should be under url

File: Integrations/integration-IronPort.yml [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+        //'Entity Value: IPv4 or IPv6 address.'
+        'mail_users_detail' : {type: 'query'},
+        //'Entity Value: Email ID of the internal user, for example, user@example.com.'
+        'mail_virus_type_detail' : {type: 'query'},
+        //'Entity Value: Name of viruses.'
+    };
+
+    var sendRequest = function(url, method) {
+        //var auth = basicAuth(params.username,params.password);
+        log('this is sendRequest');
```
Comment (Spivaka) [inline Integrations/integration-IronPort.yml]: will be written to war room no? if so remove

File: Integrations/integration-IronPort.yml [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+        var res = http(
+            url,
+            {
+                Method: method, // Can be POST, PUT, DELETE, HEAD, OPTIONS or CONNECT
+                Headers: {
+                    Accept: ['application/json'],
+                    Host: ['147.75.81.91:30009']
```
Comment (Spivaka) [inline Integrations/integration-IronPort.yml]: seems like a but here in the host.

File: Integrations/integration-IronPort.yml [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+  required: false
+- display: Port
+  name: port
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |-
+    var host = params.server.replace(/[\/]+$/, '') + ':'+params.port;
+    var commandDict = {
```
Comment (Spivaka) [inline Integrations/integration-IronPort.yml]: is it used??

File: Integrations/integration-IronPort.yml [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    var host = params.server.replace(/[\/]+$/, '') + ':'+params.port;
+    var commandDict = {
```
Comment (Spivaka) [inline Integrations/integration-IronPort.yml]: this is still here for no reason

Comment (shaniacht1) [reply Integrations/integration-IronPort.yml]: it is used in oine 110

**Pull request merged**